adam_preprocessing/singlefile_stroke_extraction.py

```diff
@@ -32,6 +32,11 @@ def main():
         default="dummy",
         help="String label to use."
     )
+    parser.add_argument(
+        "--process-masks-independently",
+        action="store_true",
+        help="If passed, will not merge adjacent masks of different colors."
+    )
```

Comment:
This arg is never passed into the stroke extraction code?

Incidentally, thanks for updating this file.